src/middlewared/middlewared/plugins/chart_releases_linux/helm.py

```diff
@@ -41,6 +42,10 @@ def helm_action(self, chart_release, chart_path, config, tn_action):
             )
             stderr = cp.communicate()[1]
             if cp.returncode:
-                raise CallError(f'Failed to {tn_action} chart release: {stderr.decode()}')
+                errmsg = stderr.decode()
+                if tn_action == 'upgrade' or tn_action == 'install':
+                    cleanupregex = re.compile(r'^Error: .+?[)]: ')
+                    errmsg = re.sub(cleanupregex, '', errmsg, 1)
```

Comment:
You can do this like `re.sub(r'^Error: .+?[)]: ', '', s, 1)`

Also i believe the regex you are looking at is `r'^Error: .+[)]?: '` (with question mark at the end)

Reply:
@stavros-k noted that the use of re- compile, was a practice used by iX in all other cases where `re.sub` was used, so we picked that for consistency.

Reply:
Regarding `+?`:
![image](https://user-images.githubusercontent.com/47820033/207443490-d7c12662-0216-4868-9b39-7b1992eedcbf.png) 

But `[)]?` It will capture also this:
`Error: INSTALLATION FAILED: execution error at (plex/templates/deployment.yaml:1:3): For example do this and that (carefully): ` 
And remove it.

TL;DR `?` can be removed and will still work

Reply:
@stavros-k either ways having it after `+` does not make sense?
Using it means safely handling closing bracket that it might not exist sometime although I assume this is always going to exist as otherwise having the colon after this does not make sense so let's just remove it in that case then.

Reply:
@Ornias1993 compiling is better but then it should be done only once after the imports by declaring it like this https://github.com/truenas/middleware/blob/c80c7bc7e091c17c0252d6c34c8abbfc18669bfb/src/middlewared/middlewared/plugins/iscsi_/iscsi_global.py#L10.
This way the regex is only compiled once and not each time the function is called..
Then you can do `RE_VARNAME.sub()`

Reply:
> @stavros-k either ways having it after `+` does not make sense? Using it means safely handling closing bracket that it might not exist sometime although I assume this is always going to exist as otherwise having the colon after this does not make sense so let's just remove it in that case then.

The only different is that it will stop "parsing" a bit faster in this case, If I'm reading in correctly.
But for the sake of readability, and not having someone scratch his header down the road, and considering the parse time is not that HUGE to make a difference.

Let's remove it.

Reply:
DOne deal!